rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/message_type_support_decl.hpp

```diff
@@ -23,7 +23,10 @@
 namespace rosidl_typesupport_introspection_cpp
 {
 
-// This is implemented in the shared library provided by this package.
+/// Returns a pointer to the type support structure provided by the
+/// lib<package_name>__rosidl_typesupport_introspection_cpp.so downstream
```

Comment:
It's not guaranteed to be `lib<package_name>`. The actual name is whatever target name was given to the CMake macro `rosidl_generate_interfaces()`. Related bug: https://github.com/ros2/rosidl_typesupport/issues/120

Reply:
Thanks, changed 7616d600cd3deb1bb83806b3d70190553aa88d73